### PR TITLE
feat: Add CI for the apisix-openresty build and install rpm

### PR DESCRIPTION
**Pull request by tzssangglass**
fix: #43

### CODE REVIEW

File: .github/workflows/package-apisix-openresty.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          docker run -itd -v /home/runner/work/apisix-build-tools/apisix-build-tools/output:/output --name centos7Instance --net="host" docker.io/centos:7 /bin/bash
+
+      - name: install dependencies in container
+        run: |
+          docker exec centos7Instance bash -c "yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo"
+          docker exec centos7Instance bash -c "yum -y install openresty-openssl111 openresty-pcre openresty-zlib"
+
+      - name: install rpm in container
+        run: |
+          docker exec centos7Instance bash -c "yum -y localinstall /output/apisix-openresty-1.0.0-0.x86_64.rpm"
```
Comment (spacewander) [inline .github/workflows/package-apisix-openresty.yml:36]: Can we run some commands to ensure openresty is installed?

Comment (tzssangglass) [reply .github/workflows/package-apisix-openresty.yml:36]: add, just run openresty use `r/usr/local/openresty/nginx/sbin/nginx`

File: .github/workflows/package-apisix-openresty.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+      - name: check and ensure openresty is installed
+        run: |
+          docker exec centos7Instance bash -c "/usr/local/openresty/nginx/sbin/nginx"
```
Comment (spacewander) [inline .github/workflows/package-apisix-openresty.yml:40]: Better to use `openresty -V` and check the output.

Comment (tzssangglass) [reply .github/workflows/package-apisix-openresty.yml]: update

**Comment by tzssangglass** [comment]
@Yiyiyimu pls review it

**Review by spacewander — APPROVED** [review]

File: .github/workflows/package-apisix-openresty.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          docker run -itd -v /home/runner/work/apisix-build-tools/apisix-build-tools/output:/output --name centos7Instance --net="host" docker.io/centos:7 /bin/bash
+
+      - name: install dependencies in container
+        run: |
+          docker exec centos7Instance bash -c "yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo"
+          docker exec centos7Instance bash -c "yum -y install openresty-openssl111 openresty-pcre openresty-zlib"
+
+      - name: install rpm in container
+        run: |
+          docker exec centos7Instance bash -c "yum -y localinstall /output/apisix-openresty-${BUILD_APISIX_OR_VERSION}-0.x86_64.rpm"
```
Comment (Yiyiyimu) [inline .github/workflows/package-apisix-openresty.yml:38]: do we need to add some checks here on whether installation succeeds

Comment (tzssangglass) [reply .github/workflows/package-apisix-openresty.yml:38]: yes, has added, take a look at below task: `check and ensure openresty is installed`

Comment (Yiyiyimu) [reply .github/workflows/package-apisix-openresty.yml:38]: Ah get it

**Review by Yiyiyimu — APPROVED** [review]

**Pull request merged**